src/is_face_detector/main.py

```diff
@@ -0,0 +1,35 @@
+from utils import load_options, get_np_image, get_pb_image
```

Comment:
change the name of the file to 'service.py' or something that is more meaningful with its function.